server/models/Group.js

```diff
@@ -248,11 +248,27 @@ export default function(Sequelize, DataTypes) {
       },
 
       addUserWithRole(user, role) {
-        return Sequelize.models.UserGroup.create({
-          role,
-          UserId: user.id,
-          GroupId: this.id
-        });
+        const lists = {};
+        lists[roles.BACKER] = 'backers';
+        lists[roles.MEMBER] = 'members';
+        lists[roles.HOST] = 'host';
+
+        const notifications = [{type:`mailinglist.${lists[role]}`}];
+
+        switch (role) {
+          case roles.HOST:
+            notifications.push({type:activities.GROUP_TRANSACTION_CREATED});
+            notifications.push({type:activities.GROUP_EXPENSE_CREATED});
```

Comment:
This should be under roles.MEMBER. MEMBER has to approve it. 

We can create a new type of activity. GROUP_EXPENSE_APPROVED and send that to the host for payment.